### PR TITLE
Add styling.

### PR DESCRIPTION
**Pull request by Muhammad0602**
### For this last milestone I have implemented the following.
- Followed the design from `Zeplin`
- Added the needed styling for the project according to the design.
- Added all missing elements and styled them accordingly.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+//     font-weight: 800;
+//     text-transform: uppercase;
+//     color: $black;
+//     opacity: 0.5;
+//   }
+
+//   .container {
+//     display: flex;
+//     gap: 2rem;
+//     width: 100%;
```
Comment (t-yanick) [inline src/App.scss]: - [ ] Kindly delete all these commented lines of code. It is not professional and you can always use git history if you want to revisit some old code.

File: src/components/BookForm.js [hunk 2]
```diff
@@ -19,10 +19,13 @@ function BookForm() {
...
-    <form onSubmit={handleSubmit}>
-      <input type="text" placeholder="Title" value={title} required onChange={(ev) => setTitle(ev.target.value)} />
-      <input type="text" placeholder="Author" value={author} required onChange={(ev) => setAuthor(ev.target.value)} />
-      <button type="submit">Add Book</button>
+    <form className="form" onSubmit={handleSubmit} action="">
+      <h2>Add new book</h2>
+      <div className="container">
+        <input className="title" type="text" placeholder="Title" value={title} required onChange={(ev) => setTitle(ev.target.value)} />
+        <input className="author" type="text" placeholder="Author" value={author} required onChange={(ev) => setAuthor(ev.target.value)} />
+        <button className="form-btn" type="submit">Add Book</button>
```
Comment (t-yanick) [inline src/components/BookForm.js:27]: - [ ] Great job working on this milestone. You have an aligment issue. As per the Zeplin design in the screenshot below, ADD BOOK button should align with the section above having the UPDATE PROGRESS button. This is not the case with your design.
Screen Shot 2023-04-15 at 3 44 32 AM

File: src/components/BookForm.js [hunk 3]
```diff
@@ -19,10 +19,13 @@ function BookForm() {
   }
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input type="text" placeholder="Title" value={title} required onChange={(ev) => setTitle(ev.target.value)} />
-      <input type="text" placeholder="Author" value={author} required onChange={(ev) => setAuthor(ev.target.value)} />
-      <button type="submit">Add Book</button>
+    <form className="form" onSubmit={handleSubmit} action="">
+      <h2>Add new book</h2>
```
Comment (t-yanick) [inline src/components/BookForm.js:23]: - [ ] In the same light, the ADD NEW BOOK text and Book title input form element should align with the section above it as you see in the screenshot below.
Screen Shot 2023-04-15 at 3 47 43 AM

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Muhammad0602 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Linter checks all passing

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.scss (hunk 1 above)
Comment (Muhammad0602) [reply src/App.scss]: ooh I forgot to remove these, thanks @t-yanick.

File: src/components/BookForm.js (hunk 3 above)
Comment (Muhammad0602) [reply src/components/BookForm.js:23]: thanks for your kind suggestions @t-yanick , I will do my best to implement all of these changes.

**Review by franclobo — APPROVED** [review]
Hi @Muhammad0602, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️
- Descriptive PR ✔️
- Correct Git flow ✔️
- Good UI ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**